### PR TITLE
Fix small problems

### PR DESCRIPTION
**Pull request by tanin47**
* Show job and release on the CI error message
* Raise UnreproducibleError when it cannot reproduce, so that we can see the error message on the log. Right now we merely see `SystemExit`.

### CODE REVIEW

File: ci/continuous_integration/daemon/main.py [hunk 1]
```diff
@@ -112,24 +116,35 @@ def load_new_testcases():
     auth_header = f.read()
 
   testcases = []
+  testcase_ids = set()
   page = 1
   supported_jobtypes = get_supported_jobtypes()
 
-  def _validate_testcase(testcase):
+  def _valid(testcase):
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: s/_valid/_is_valid ?

File: ci/continuous_integration/daemon/main.py [hunk 2]
```diff
@@ -112,24 +116,35 @@ def load_new_testcases():
...
     """Filter by jobtype and whether it has been successfully reproduced."""
-    return (testcase['jobType'] in supported_jobtypes['chromium'] and not
-            (testcase['id'] in TESTCASE_CACHE and
-             TESTCASE_CACHE[testcase['id']]))
+    return (testcase['jobType'] in supported_jobtypes['chromium'] and
+            not testcase['id'] in TESTCASE_CACHE and
+            not testcase['id'] in testcase_ids)
 
   while len(testcases) < 40:
+    print page
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: Do we need this print.

File: ci/continuous_integration/daemon/main.py [hunk 3]
```diff
@@ -112,24 +116,35 @@ def load_new_testcases():
...
     r = requests.post('https://clusterfuzz.com/v2/testcases/load',
                       headers={'Authorization': auth_header},
                       json={'page': page, 'reproducible': 'yes'})
-    new_testcases = r.json()['items']
-    new_testcases = [testcase['id'] for testcase in new_testcases
-                     if _validate_testcase(testcase)]
-    testcases.extend(t for t in new_testcases if t not in testcases)
+
+    empty = True
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: s/empty/found_testcases  or something ?

File: ci/continuous_integration/daemon/main.py [hunk 4]
```diff
@@ -154,7 +169,7 @@ def build_master_and_get_version():
   shutil.copy(os.path.join(TOOL_SOURCE, 'dist', 'clusterfuzz-ci.pex'),
               BINARY_LOCATION)
 
-  return call('git rev-parse HEAD', capture=True, cwd=TOOL_SOURCE).strip()
+  return call('git rev-parse HEAD', capture=True, cwd=TOOL_SOURCE).strip()[:7]
```
Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: Why :7, probably do len(string)

File: ci/continuous_integration/daemon/main.py (hunk 1 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Done.

File: ci/continuous_integration/daemon/main.py (hunk 2 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Oops added by mistake.

File: ci/continuous_integration/daemon/main.py (hunk 3 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: I changed it to `has_valid_testcase`.

File: ci/continuous_integration/daemon/main.py (hunk 4 above)
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Showing the full SHA on the log is too long... So, we only show the first 7 characters.

Comment (inferno-chromium) [inline ci/continuous_integration/daemon/main.py]: Add a comment then.

Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: Done. I explained why we use the first 7 characters.

**Pull request merged**